routes/transaction.go

```diff
@@ -3471,6 +3485,29 @@ func TransactionSpendingLimitToResponse(
 			transactionSpendingLimitResponse.DAOCoinLimitOrderLimitMap[buyingPublicKey][sellingPublicKey] = opCount
 		}
 	}
+
+	if len(transactionSpendingLimit.AssociationLimitMap) > 0 {
+		for associationLimitKey, opCount := range transactionSpendingLimit.AssociationLimitMap {
+			associationClassString := associationLimitKey.AssociationClass.ToAssociationClassString()
+			associationType := associationLimitKey.AssociationType
+			associationAppScopeTypeString := associationLimitKey.AppScopeType.ToAssociationAppScopeTypeString()
+			associationOperationString := associationLimitKey.Operation.ToAssociationOperationString()
+			var appPublicKey string
+			if !associationLimitKey.AppPKID.IsZeroPKID() {
```

Comment:
If the AppPKID is the ZeroPKID, don't we want to set the appPublicKey to the ZeroPublicKey? Or you're thinking we'll leave it as the empty string?

Reply:
I guess this depends on the `AppScopeType`, right? so we want to differentiate between the global app public key and the "any" app public key (which would be represented by the empty string)